TrkDiag/src/BkgDiag_module.cc

```diff
@@ -300,9 +309,17 @@ namespace mu2e
       _nactive = _nstereo = _nsactive = _nbkg = 0;
       bool pce = _pgen==2; // primary from a CE
       _nindex=0;
+      double sumEdep(0.);
+      double sqrSumDeltaTime(0.);
+      double sqrSumDeltaX(0.);
+      double sqrSumDeltaY(0.);
       for(auto const& ich : cluster.hits()){
         ComboHit const& ch = _chcol->at(ich);
         BkgClusterHit const& bhit = _bkghitcol->at(ich);
+        sumEdep +=  ch.energyDep();
+        sqrSumDeltaX += std::pow(ch.pos().X() - _cpos.X(),2);
```

Comment:
The RMS calculation is mathematically correct, but I suspect we will have better discrimination computing the rms of the transverse distance to the straw, not the hit.  That will remove the uncertainty from the position along the straw, and it automatically combines the 2 dimensions.  You can compute that using 2 line POCA, one line being in Z through the cluster position, the other being along the straw (udir) through the hit position.  Or its simple enough to compute analytically.